DESCRIPTION

```diff
@@ -23,7 +23,7 @@ Description: Create interactive web graphics from 'ggplot2' graphs and/or a cust
 URL: https://plotly-r.com, https://github.com/plotly/plotly.R, https://plotly.com/r/
 BugReports: https://github.com/plotly/plotly.R/issues
 Depends:
-    R (>= 3.2.0),
+    R (>= 3.5),
```

Comment:
```suggestion
    R (>= 3.2.0),
```

Reply:
Addressed in https://github.com/plotly/plotly.R/pull/2336/commits/93ef777a672852a2aa48c9f7ed5c4c2f995713b5.